### PR TITLE
test(dropdown): integrated and added test case for a component

### PR DESCRIPTION
**Pull request by theJohnnyMe**
- Adding regular dropdown to the form page
- Demoing extraction of value from it

### CODE REVIEW

**Comment by mJarsater** [comment]
Do you want to add the dropdown-v2 also?

File: src/pages/FormPage.tsx [hunk 1]
```diff
@@ -12,159 +13,186 @@ const FormPage = () => {
   const handleSubmit = (event: FormEvent) => {
     event.preventDefault();
     if (form.current) {
+      const countryDropdownValue =
+        countryDropdown.current?.getAttribute("selected-value");
```
Comment (mJarsater) [inline src/pages/FormPage.tsx:17]: does countryDropdown.current?.selectedValue not work? Just curious

Comment (theJohnnyMe) [reply src/pages/FormPage.tsx:17]: Good catch, let me try it out. I thought of similar once I saw the dropdown-v2 code example and passing "options"...

Comment (mJarsater) [reply src/pages/FormPage.tsx:17]: Ah alright!

**Review by mJarsater — APPROVED** [review]
LGTM, maybe we shouldn't spend to much time on the "old" dropdown

**Pull request merged**